web/src/features/fba/components/viz/ElevationFlag.tsx

```diff
@@ -1,40 +1,30 @@
 import React from 'react'
-import { Box, Grid, Typography } from '@mui/material'
-
-const FLAG_COLOUR = '#CCCCCC'
+import { Typography } from '@mui/material'
+import Grid from '@mui/material/Unstable_Grid2'
+import Flag from '@/features/fba/components/viz/FillableFlag'
 
 interface ElevationFlagProps {
-  testId?: string
+  id: string
   percent: number
+  testId?: string
 }
 
-const ElevationFlag = ({ percent, testId }: ElevationFlagProps) => {
+const ElevationFlag = ({ id, percent, testId }: ElevationFlagProps) => {
   return (
-    <Grid item sx={{ display: 'flex', alignItems: 'center', flexGrow: 1, justifyContent: 'flex-start' }} xs={12}>
-      <Box
+    <Grid sx={{ alignItems: 'center', display: 'flex', justifyContent: 'flex-end' }} xs={6}>
```

Comment:
Should this still be a Grid `item`?

Reply:
Not anymore! With the new grid implementation all grids are considered `item`s by default. Parent grids must be marked with a `container` prop now.